src/core/init/initialize_media_source.ts

```diff
@@ -291,7 +268,7 @@ export default function InitializeOnMediaSource(
       });
 
       // handle initial load and reloads
-      const recursiveLoad$ = recursivelyLoadOnMediaSource(initialMediaSource,
+      const recursiveLoad$ = recursivelyLoadOnMediaSource(initMediaSource,
```

Comment:
initial was a better name, no?

Here it makes me think we're going to initialize it...